docs/widgets/input.md

```diff
@@ -20,10 +22,52 @@ The example below shows how you might create a simple form using two `Input` wid
     --8<-- "docs/examples/widgets/input.py"
     ```
 
+### Validating Input
+
+You can supply one or more *validators* to the `Input` widget to validate the value.
```

Comment:
Can you link to the Validator base class here?

Reply:
Done.